kafka-connect-s3/src/test/java/io/confluent/connect/s3/TestWithMockedS3.java

```diff
@@ -81,7 +81,7 @@ public void setUp() throws Exception {
   @Override
   public void tearDown() throws Exception {
     super.tearDown();
-    s3mock.stop();
+    s3mock.shutdown(); // shutdown the Akka and HTTP frameworks to close all connections
```

Comment:
This is the one line that we need to fix the Java 11 issue, and `shutdown()` was introduced in 0.2.5. All other changes in this PR address behavioral changes in the newer S3Mock version.